CONTRIBUTING.md

```diff
@@ -9,20 +9,29 @@ Melange can be set up with [#nix](Nix) or [#esy](Esy). Instructions for both are
 
 ### Nix
 
-Obtaining the [Nix](https://nixos.org/) package manager is a prerequisite.
+**Prerequisites**:
+- Obtaining the [Nix](https://nixos.org/) package manager
+- Installing [Nix flakes](https://nixos.wiki/wiki/Flakes)
 
 
 The best way to get started is to get a `nix` shell running:
 
 ```sh
 # Opens a shell with the necessary environment.
-nix-shell --pure
```

Comment:
Ah, I see where you got `nix-shell` from now. Yeah we stopped supporting this workflow a while ago. Flakes feel easier to me.